src/main/java/com/easyrest/facade/WaiterOperationsFacade.java

```diff
@@ -0,0 +1,68 @@
+package com.easyrest.facade;
+
+import com.easyrest.components.waiter.OrderAction;
+import com.easyrest.components.waiter.WaiterMenu;
+import com.easyrest.pages.WaiterPage;
+import com.easyrest.pages.waiterPanel.AssignedWaiterOrders;
+import com.easyrest.pages.waiterPanel.InProgressOrders;
+import org.openqa.selenium.WebDriver;
+
+public class WaiterOperationsFacade {
+    private final WaiterPage waiterPage;
+    private final OrderAction orderAction;
+
+    public WaiterOperationsFacade(WebDriver driver) {
+        this.waiterPage = new WaiterPage(driver);
+        this.orderAction = new OrderAction(driver);
+    }
+
+    public void seeAllOrders() {
+        goToTabMenu()
+                .clickOnAllOrders();
+    }
+
+    public AssignedWaiterOrders seeAssignedWaiterOrders() {
+        return goToTabMenu()
+                .clickOnAssignedWaiterOrders();
+    }
+
+    public InProgressOrders seeInProgressOrders() {
+        return goToTabMenu()
+                .clickOnInProgressOrders();
+    }
+
+    public void seeOrdersHistory() {
+        goToTabMenu()
+                .clickOnOrdersHistory();
+    }
+
+    public Integer seeAssignedWaiterOrdersCount() {
+        return seeAssignedWaiterOrders()
+                .getOrdersCount();
+    }
+
+    public Integer seeInProgressOrdersCount() {
+        return seeInProgressOrders()
+                .getOrdersCount();
+    }
+
+    public void startOrder() {
+        seeAssignedWaiterOrders()
+                .getOrder()
+                .clickOnExpandOrderDetails();
+        orderAction.clickOnOrderAction();
+    }
+
+    public void closeOrder() {
+        seeInProgressOrders()
+                .getOrder()
+                .clickOnExpandOrderDetails();
+        orderAction.clickOnOrderAction();
+    }
+
+    private WaiterMenu goToTabMenu() {
```

Comment:
Private method should be higher

Reply:
Oracle: these methods should be grouped by functionality rather than by scope or accessibility. For example, a private class method can be in between two public instance methods. The goal is to make reading and understanding the code easier.

Reply:
Mmm, okey)